src/main/webapp/app/shared/components/listing-card/listing-card.tsx

```diff
@@ -0,0 +1,38 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React from 'react';
+import { Row, Col } from 'reactstrap';
+import './listing-card.scss';
+
+export interface IListingCardProps {
+  showActionMenu: boolean;
+  actionMenuHandler?: Function;
+  title?: string;
+}
+
+export class ListingCard extends React.Component<IListingCardProps> {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <div className="card-container container">
+        <Row className="align-items-center justify-content-center">
+          <Col xs="10">
+            {this.props.title && this.props.title.hasOwnProperty('value') ? (
+              <span className="font-weight-bold text-dark d-block mb-1">{this.props.title}</span>
+            ) : null}
+            {this.props.children}
+          </Col>
+          <Col xs="1">
+            {this.props.showActionMenu ? (
+              <span onClick={this.props.actionMenuHandler()} className="hand">
```

Comment:
`onClick={this.props.actionMenuHandler()}`
you should not include the paranthesis. this is a bug. the correct syntax is 
```
              <span onClick={this.props.actionMenuHandler} className="hand">
```

if you include the `()`, you are asking the code to immediately invoking the function, even the user did not click anything.

resource: https://reactjs.org/docs/faq-functions.html#why-is-my-function-being-called-every-time-the-component-renders

Reply:
Update in f2131ff5e4d6687aed3eb5beb220f1dc9cce08f3